packages/node/src/node-types.ts

```diff
@@ -0,0 +1,8 @@
+import { types } from "@counterfactual/cf.js";
+
+export import MethodRequest = types.Node.MethodRequest;
+export import MethodResponse = types.Node.MethodResponse;
+export import GetAppInstancesResult = types.Node.GetAppInstancesResult;
+export import Address = types.Address;
+export import AppInstanceInfo = types.AppInstanceInfo;
+export import MethodName = types.Node.MethodName;
```

Comment:
why not use typescript-typings

Reply:
That's my task to move all common types to typescript-typings. Will be my next goal after #288